package.json

```diff
@@ -26,17 +26,14 @@
     "@babel/eslint-parser": "^7.12.1",
     "@octokit/rest": "^18.0.12",
     "@vue/cli": "^4.5.9",
-    "@vue/cli-plugin-babel": "^4.5.9",
-    "@vue/cli-plugin-eslint": "^4.5.9",
-    "@vue/cli-service": "^4.5.9",
-    "@vue/compiler-sfc": "^3.0.4",
+    "@vue/cli-plugin-babel": "^4.5.10",
+    "@vue/cli-plugin-eslint": "^5.0.0-alpha.2",
+    "@vue/cli-service": "^4.5.10",
     "electron": "^10.2.0",
-    "electron-devtools-installer": "^3.1.0",
-    "eslint": "^6.7.2",
-    "eslint-plugin-vue": "^7.2.0",
-    "follow-redirects": "^1.13.1",
```

Comment:
`follow-redirects` is used as part of the setup script and should not be removed... https://github.com/mattfarina/rd/blob/5d6433be7601aa5eb0c42d27a56e3347c582206f/scripts/setupmac.js#L1

Reply:
Thanks, fixed in 1eded433e3c7af1370f2a1ace24e9a94cf3cd4e8 — without this, no errors were thrown but `setupmac` just hangs. Fun!